config/reports/ukraineSubsidyOproep1Report.js

```diff
@@ -0,0 +1,137 @@
+import { generateReportFromData, batchedQuery } from "../helpers.js";
+import { getSafeValue } from "./util/report-helpers";
+
+export default {
+  cronPattern: "0 0 0 * * *",
```

Comment:
We might have preferred another moment as all GzG rapports also get generated at midnight. We prefer to spread the load a bit. perhaps this might be better `0 30 23 * * *` 23h30

Reply:
Did not know this, I will change it.